src/handlers/ServerCommunicator.ts

```diff
@@ -107,8 +110,122 @@ export class ServerCommunicator {
         router.delete('/play/bus', (req:Request, res:Response)=>{
             this.gameplayHandler.handle(req, res);
         })
-        router.get('/play', (req:Request, res:Response)=>{
-            this.gameplayHandler.handle(req, res);
+        router.get('/play/:id', (req:Request, res:Response)=>{
+            console.log(req.params.id);
+            let routeCards = [
```

Comment:
Once Michael's changes are ready we should make this so we shuffle the route cards and deal three to each player instead of giving the same ones to each player. This is a good start though.